src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -247,18 +248,18 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
     return severityMap.normal;
   }
 
-  private _translateTurn(value: number, config: Config) {
+  private _translateTurn(value: number, config: Config): number {
     const maxTurnValue = Math.min(Math.max(value, config.min!), config.max!);
     return (
       (5 * (maxTurnValue - config.min!)) / (config.max! - config.min!) / 10
     );
   }
 
-  private _computeBaseUnit() {
+  private _computeBaseUnit(): string {
     return this.clientWidth < 200 ? this.clientWidth / 5 + "px" : "50px";
   }
 
-  private _handleClick() {
+  private _handleClick(): void {
     fireEvent(this, "hass-more-info", { entityId: this._config!.entity });
   }
 }
```

Comment:
This does not have a `HTMLElementTagNameMap` global interface defined

Reply:
Good catch

Reply:
I had this pretty much same PR on my list of things to TODO so already knew about it 😄